pkg/app/ops/handler/handler.go

```diff
@@ -164,9 +164,10 @@ func (h *Handler) handleAddProject(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var (
-		id            = r.FormValue("ID")
-		description   = r.FormValue("Description")
-		sharedSSOName = r.FormValue("SharedSSO")
+		id                  = r.FormValue("ID")
+		description         = r.FormValue("Description")
+		sharedSSOName       = r.FormValue("SharedSSO")
+		viewerRoleAsDefault = r.FormValue("ViewerRoleAsDefault") == "on"
```

Comment:
Fix this name too.

Reply:
Nice catch, addressed on https://github.com/pipe-cd/pipe/pull/2370/commits/b9eb3fecadb5eeb2f35e639d334a45677f539e71 🙏